airbyte-integrations/connectors/source-mongodb/lib/mongodb_configured_stream/incremental.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../airbyte_logger.rb'
+
+require_relative './base.rb'
+require_relative '../mongodb_types_converter.rb'
+
+class MongodbConfiguredStream::Incremental < MongodbConfiguredStream::Base
+  def new(configured_stream:, state:)
+    super
+
+    value = @state.get(stream_name: stream_name, cursor_field: cursor_field)
+    @cursor =  MongodbTypesConverter.convert_value_to_type(value, stream['json_schema']['properties'][cursor_field]['type'])
+  end
+
+  def cursor_field
+    @cursor_field ||= configured_stream['cursor_field']&.first
```

Comment:
the docs should indicate that the cursor field can't be nested (the Airbyte protocol allows nested cursor fields, that's why the field is an array)

Reply:
agree. added to the docs